build.boot

```diff
@@ -31,13 +31,13 @@
 (bootlaces! +version+)
 
 (task-options!
-  javac   {:options (let [jdk7-bootclasspath (System/getenv "JDK7_BOOTCLASSPATH")]
+  javac   {:options (let [jdk8-bootclasspath (System/getenv "JDK7_BOOTCLASSPATH")]
                       (concat
                         []
-                        (when-not (empty? jdk7-bootclasspath)
-                          ["-source" "1.7"
-                           "-target" "1.7"
-                           "-bootclasspath" jdk7-bootclasspath])))}
+                        (when-not (empty? jdk8-bootclasspath)
+                          ["-source" "1.8"
+                           "-target" "1.8"
+                           "-bootclasspath" jdk8-bootclasspath])))}
```

Comment:
I think we'll be able to just remove this and use `["-source" "1.8" "-target" "1.8"]` as the `javac` options. This worked for me, I assume because I have JDK 1.8 installed as the default. We could always do a similar thing in the future when Java 9 becomes more ubiquitous.